addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14266,13 +14276,13 @@ msgstr ""
 #. Description of setting "Videos -> Library -> Flatten TV show seasons" with label #20412
 #: system/settings/settings.xml
 msgctxt "#36144"
-msgid "Remove the TV show season node, toggles between 'If only one season' (default), 'Always' and 'Never'."
+msgid "Remove the TV show season node, toggles between \"If only one season\" (default), \"Always\" and \"Never\"."
```

Comment:
When doing the help description last year I only used square brackets where each of the options needed to be individually described, so 

[Setting option 1] option 1 description [Setting option 2] option 2 description

I did this way since to allow the eye to more clearly pick out the relevant bit of text, since we couldn't use formatting such as bold or italics.